myskoda/models/spin.py

```diff
@@ -0,0 +1,25 @@
+"""Models for responses of api/v2/garage/vehicles/{vin}."""
+
+import logging
+from dataclasses import dataclass, field
+from enum import StrEnum
+
+from mashumaro.mixins.orjson import DataClassORJSONMixin
+
+_LOGGER = logging.getLogger(__name__)
+
+class VerificationStatus(StrEnum):
+    """List of known statuses for SPIN."""
+    CORRECT_SPIN = "CORRECT_SPIN"
+    INCORRECT_SPIN = "INCORRECT_SPIN"
+
+@dataclass
+class SpinStatus(DataClassORJSONMixin):
+    state: str
+    remainingTries: int
+    lockedWaitingTimeInSeconds: int
+
+@dataclass
+class Spin(DataClassORJSONMixin):
+    verificationStatus: VerificationStatus
+    spinStatus: SpinStatus | None = field(default=None)
```

Comment:
Please use `snake_case` for class attributes and and alias in `field_options` to match the field name in the API response. E.g.

```suggestion
@dataclass
class SpinStatus(DataClassORJSONMixin):
    state: str
    remaining_tries: int = field(metadata=field_options(alias="remainingTries"))
    locked_waiting_time_in_seconds: int = field(metadata=field_options(alias="lockedWaitingTimeInSeconds"))

@dataclass
class Spin(DataClassORJSONMixin):
    verification_status: VerificationStatus = field(metadata=field_options(alias="verificationStatus"))
    spin_status: SpinStatus | None = field(default=None, metadata=field_options(alias="verificationSspinStatustatus"))
```